web/api/web_api_v1.c

```diff
@@ -778,6 +778,8 @@ static inline void web_client_api_request_v1_info_mirrored_hosts(BUFFER *wb) {
     int count = 0;
     rrd_rdlock();
     rrdhost_foreach_read(rc) {
+        if (rrdhost_flag_check(rc, RRDHOST_FLAG_ARCHIVED))
+            continue;
```

Comment:
not sure this is needed. When straming host goes offline we still list it anyway? With MVP1 we actually add `reachable` flag that is based on `rc->connected_senders > 0`

Reply:
It's about the dashboard not showing all archived hosts in the top left corner, because they are empty.

Reply:
yes but wouldn't that be same situation when child goes offline? Yet it still is in that list.

Reply:
It's unreachable, not archived. This is the same even today, you can actually lookup the data. If you restart the agent it will be archived.

Reply:
the list seems to me before this PR as "Any child node that ever connected to agent" but it gets lost after an agent restarts.

Wouldn't it be better after this PR "Any child node that ever connected to agent" but now we actually keep history between restarts?

Anyway, I will not block PR on this comment so I mark it resolved.